jest.config.js

```diff
@@ -1,15 +1,16 @@
-const path = require('path')
+const path = require("path");
+
+const pathAsArr = process.env.PNPM_SCRIPT_SRC_DIR.split(path.sep);
+const packageName = pathAsArr[pathAsArr.length - 1];
```

Comment:
maybe read the PNPM_PACKAGE_NAME env variable instead

Reply:
I don't see this variable. Only `npm_package_name`